Library/Homebrew/diagnostic.rb

```diff
@@ -811,6 +813,18 @@ def check_coretap_git_origin
               git -C "#{coretap_path}" remote set-url origin #{Formatter.url(remote)}
           EOS
         end
+
+        return if ENV["CI"] || ENV["JENKINS_HOME"]
+
+        branch = coretap_path.git_branch
+        return if branch.nil? || branch =~ /master/
+
+        <<-EOS.undent
+          Homebrew/homebrew-core is not on the master branch
```

Comment:
I think `Homebrew` has to be replaced with a variable that == `Linuxbrew` on Linux
@sjackman

Reply:
I'm fine with this as is. We refer to this tap as `Homebrew/core` even when it's our Linuxbrew fork.